cookbooks/aws-parallelcluster-slurm/files/default/config_slurm/scripts/health_check_manager.py

```diff
@@ -252,6 +261,9 @@ def _execute_health_checks(health_check_manager_config: HealthCheckManagerConfig
                 else:
                     output = " empty"
                 log.info("Output of Health Check '%s' execution is%s", health_check.name, output)
+                publish_health_check_result(
```

Comment:
are you handling "publish_health_check_result" expections?

Reply:
I was protected the inner base publish_event call, but not the outer call. I've update the code so the outer call is guarded as well.